app/_how-tos/allow-multiple-authentication.md

```diff
@@ -26,14 +27,41 @@ entities:
   - consumer
 
 tags:
-    - authentication
+  - authentication
 
 tldr:
-    q: Placeholder
-    a: Placeholder
+  q: How do I allow different clients to access an upstream service with different authentication types, and forbid access to any unauthenticated clients?
+  a: |
+    You can use multiple authentication plugins with an anonymous Consumer to give clients multiple options for authentication. 
+    The anonymous Consumer acts as a fallback to catch all other unauthorized requests.
 
+    For example, you can configure Key Auth and Basic Auth, apply them to specific Consumers, and set `anonymous` in those plugins to catch access attempts from anyone else.
+    Then, apply the Request Termination plugin to requests made with the anonymous Consumer to terminate the requests and send back a specific message.
```

Comment:
```suggestion
  a: |
    Configure multiple authentication plugins, like [Key Auth](/plugins/key-auth/) and [Basic Auth](/plugins/basic-auth/), and apply them to specific [Consumers](/gateway/entities/consumer/). Set `config.anonymous` in those plugins to the ID of the anonymous Consumer to catch access attempts from anyone else.
    Then, apply the [Request Termination](/plugins/request-termination/) plugin to requests made with the anonymous Consumer to terminate the requests and send back a specific message.
```
Moving the "intro" content to the steps and added links so someone could just look at the TLDR and know how to configure this without reading the steps.